Mathlib/Algebra/Star/Subalgebra.lean

```diff
@@ -792,6 +792,20 @@ protected def codRestrict (f : A →⋆ₐ[R] B) (S : StarSubalgebra R B) (hf :
   toAlgHom := AlgHom.codRestrict f.toAlgHom S.toSubalgebra hf
   map_star' := fun x => Subtype.ext (map_star f x)
 
+@[simp]
+theorem subtype_comp_codRestrict (f : A →⋆ₐ[R] B) (S : StarSubalgebra R B)
+    (hf : ∀ x : A, f x ∈ S) : S.subtype.comp (f.codRestrict S hf) = f :=
+  StarAlgHom.ext fun _ => rfl
```

Comment:
nit: move this below `codRestrict_apply` as the `fun _ => rfl` here is `codRestrict_apply _ _ _`